CHANGELOG.rst

```diff
@@ -2,6 +2,14 @@
 Changelog
 =========
 
+Unreleased
+----------
+
+* Implement closed layers - a feature that prevents imports from higher layers
```

Comment:
Could we change this to `Add closed layers to layer contract.` - and skip the rest?